python/ray/dashboard/client/src/pages/dashboard/Tune.tsx

```diff
@@ -0,0 +1,125 @@
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import createStyles from "@material-ui/core/styles/createStyles";
+import withStyles, { WithStyles } from "@material-ui/core/styles/withStyles";
+import React from "react";
+import { connect } from "react-redux";
+import { getTuneInfo } from "../../api";
+import { StoreState } from "../../store";
+import { dashboardActions } from "./state";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      padding: theme.spacing(2),
+      "& > :not(:first-child)": {
+        marginTop: theme.spacing(2)
+      }
+    },
+    table: {
+      marginTop: theme.spacing(1)
+    },
+    cell: {
+      padding: theme.spacing(1),
+      textAlign: "center",
+      "&:last-child": {
+        paddingRight: theme.spacing(1)
+      }
+    }
+  });
+
+const mapStateToProps = (state: StoreState) => ({
+  tuneInfo: state.dashboard.tuneInfo,
+});
+
+const mapDispatchToProps = dashboardActions;
+
+class Tune extends React.Component<WithStyles<typeof styles> &
+  ReturnType<typeof mapStateToProps> &
+  typeof mapDispatchToProps> {
+  interval: number = 0;
+
+  refreshTuneInfo = async () => {
+    try {
+      const [tuneInfo] = await Promise.all([
+        getTuneInfo()
+      ]);
+      this.props.setTuneInfo({ tuneInfo });
+    } catch (error) {
+      this.props.setError(error.toString());
+    }
+  };
+
+  async componentDidMount() {
+    this.interval = window.setInterval(this.refreshTuneInfo, 1000);
```

Comment:
Can we instead do what we're doing elsewhere where we `setTimeout` inside the refresh? If one of these requests takes longer than a second due to a large return value / network latency, then `setInterval` will start queueing up more and more of these calls over time without waiting for the backlog to clear out.